code/modules/client/preference_setup/loadout/loadout_utility.dm

```diff
@@ -9,10 +9,19 @@
 	path = /obj/item/weapon/clipboard
 
 /datum/gear/utility/communicator
-	display_name = "personal communicator"
+	display_name = "communicator selection"
 	path = /obj/item/device/communicator
 	cost = 0
 
+	display_name = "communicator watch"
+/datum/gear/utility/communicator/New()
+	..()
```

Comment:
Should mention in PR changes this is a thing, since it alters the loadout choices and people will need to reselect it probably?